### DISCUSSION TITLE
Custom metric for attribute

### DISCUSSION THREAD

**Discussion by ars3nicum** [saleor/saleor#14072] [Q&A] [answered]
Is it possible to add millimetres metric to an attribute of a Numeric type?

**Comment by peteror** [comment]
As I see, the metrics are set in the Saleor code - it would be quite easy to modify the code, but you'd need to build your own docker images to use it.

**Comment by timuric** [comment, marked as answer]
@ars3nicum yes millimeters and many other measures are supported in attributes

**Reply by ars3nicum** [reply]
But I have only meters, centimeters and kilometers for Distance unit.
How to activate others? @timuric

**Reply by ars3nicum** [reply]
same with demo dashboard at https://demo.saleor.io/

**Reply by timuric** [reply]
This PR addresses this issue https://github.com/saleor/saleor-dashboard/pull/4402
